From 7c7c51037b386b23587fa76789c217b19be1e8de Mon Sep 17 00:00:00 2001
From: Guo Mengqi <guomengqi3@huawei.com>
Date: Sat, 26 Aug 2023 09:59:53 +0800
Subject: [PATCH 2/2] arch: arm64: Add config for smmu queue size customizing

Offering: HULK
ascend inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I7VD6D

--------------------------------

Previous commit i0d72547335c8 ("[Huawei] iommu/arm-smmu-v3: Provide
a module parameter for changing smmu queue length") adds smmu queue size
customizing ability for ascend.

Now add relevant config to Kconfig file.

Signed-off-by: Guo Mengqi <guomengqi3@huawei.com>
---
 arch/arm64/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index a767720..abe2972 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -2195,6 +2195,13 @@ config ASCEND_BIND_ROOTFS_ON_NORMAL
 	  This feature bind rootfs to regular memory nodes(movable nodes are excluded),
 	  MEMORY_HOTPLUG is necessary as it is required by the feature movable nodes.
 
+config ASCEND_SMMU_QUEUE_SIZE
+    bool "Enable support for changing smmu queue size"
+    default n
+    depends on ARM_SMMU_V3
+    help
+      This option enables user to customize smmu queue size via a module param.
+
 endif
 
 endmenu
-- 
2.17.1

